### PR TITLE
Tardies shown as number rather than percent

### PR DESCRIPTION
**Pull request by RobertAKARobin**
- TODO: Add coloring
- Changes % absent to % present
- Fixes https://github.com/ga-dc/garnet/issues/524

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Fixes https://github.com/ga-dc/garnet/issues/533

File: app/views/memberships/_table.html.erb [hunk 1]
```diff
@@ -20,7 +20,7 @@
     </thead>
     <tbody>
       <% memberships.each do |membership| %>
-        <%= render partial: "memberships/row", locals: {membership: membership, row_header: row_header} %>
+        <%= render partial: "memberships/row", locals: {membership: membership} %>
```
Comment (jshawl) [inline app/views/memberships/_table.html.erb]: what's the row header?

**Comment by jshawl** [comment]
:+1:

File: app/views/memberships/_table.html.erb (hunk 1 above)
Comment (RobertAKARobin) [inline app/views/memberships/_table.html.erb]: No idea

**Comment by amaseda** [comment]
:+1:

**Comment by RobertAKARobin** [comment]
Verbal thumb from Adrian

**Comment by RobertAKARobin** [comment]
Actual thumb from Adrian

**Pull request merged**